tests/unit/dataactvalidator/test_d32_detached_award_financial_assistance_1.py

```diff
@@ -0,0 +1,28 @@
+from tests.unit.dataactcore.factories.staging import DetachedAwardFinancialAssistanceFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'd32_detached_award_financial_assistance_1'
+
+
+def test_column_headers(database):
+    expected_subset = {"row_number", "period_of_performance_star"}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
+def test_success(database):
+    """ PeriodOfPerformanceStartDate is an optional field, but when provided, must follow YYYYMMDD format """
+    det_award_1 = DetachedAwardFinancialAssistanceFactory(period_of_performance_star="19990131")
```

Comment:
we should probably add a test case for a null value to make sure it's omitted. And if we decide to avoid blanks, a blank test case should also be added.